src/devices/wifi/nm-device-iwd.c

```diff
@@ -244,6 +244,11 @@ insert_ap_from_network (GHashTable *aps, const char *path, int16_t signal, uint3
 	uint8_t bssid[6];
 	NMWifiAP *ap;
 
+	if (g_hash_table_lookup (aps, path)) {
+		nm_log_dbg (LOGD_WIFI, "Duplicate network at %s", path);
```

Comment:
use `_LOGD()` macro for logging. Yes, this requires a `@self` pointer , but that is a reasonable requirement (as it gives you the necessary context for logging).